src/components/molecules/ChatMessageBox/ChatMessageBox.scss

```diff
@@ -13,6 +13,11 @@
 
   &__input {
     width: 100%;
+
+    .InputField__input {
```

Comment:
Why is this class being added here? Presumably to override some centreing happening in this file or similar? This feels like there is possibly a better solution

Reply:
@0xdevalias 

<img width="313" alt="Screenshot 2021-07-16 at 16 03 40" src="https://user-images.githubusercontent.com/26366773/125952248-c6d8a4ed-8464-4a4c-9a39-9f50e6449044.png">

I'm overriding the globals styles, adding the relevant indent, and fixing directly for this Chat input. WDYT?